core/server/services/themes/index.js

```diff
@@ -83,8 +83,22 @@ module.exports = {
         // no need to check the score, activation should be used in combination with validate.check
         // Use the two theme objects to set the current active theme
         try {
+            let previousGhostAPI;
+
+            if (this.getActive()) {
```

Comment:
We ask if we have an active theme, but if not, we just call `this.getActive()` 🤔 
See line 93.

Reply:
Yes, but there's:
```
active.set(loadedTheme, checkedTheme, error);
```
between these two calls, which changes the context a bit. `previousGhostAPI` would be `undefined` when starting up the instance first time ;)

Reply:
Ah yeah true 👍